docs/src/pages/components/about-the-lab/about-the-lab.md

```diff
@@ -36,3 +36,21 @@ npm install @material-ui/core
 // with yarn
 yarn add @material-ui/core
 ```
+
+## TypeScript
+
+In order to benefit from the [CSS overrides](/customization/globals/#css) and [default prop customization](/customization/globals/#default-props) with the theme, TypeScript users need to use [module augmentation](/guides/typescript/#customization-of-theme) to extend the default theme structure with the extension components available in the lab:
```

Comment:
Maybe mention that we do the module augmentation for them and they just need to include it?

Reply:
```suggestion
In order to benefit from the [CSS overrides](/customization/globals/#css) and [default prop customization](/customization/globals/#default-props) with the theme, TypeScript users need to import the following types. Internally, it uses [module augmentation](/guides/typescript/#customization-of-theme) to extend the default theme structure with the extension components available in the lab.
```